src/playercards/PlayerCardSpawner.ttslua

```diff
@@ -82,7 +82,7 @@ end
 -- @param pos Position table where the cards should be spawned (global)
 -- @param rot Rotation table for the orientation of the spawned cards (global)
 -- @param callback Function, callback to be called after the card/deck spawns.
-Spawner.spawn = function(cardList, pos, rot, callback)
+Spawner.spawn = function(cardList, pos, rot, callback, sidewaysDeck)
```

Comment:
Is there a way we can inspect the cards in the pile rather than relying on a passed parameter?